### ISSUE TITLE
pull changes from SAP/cf-python-logging-support add support for Django and Falcon

### ISSUE THREAD

**Issue by thangbn** [thangbn/json-logging-python#27]
https://github.com/SAP/cf-python-logging-support have a quite similar setup in term of adding framework support.  We can pull Django and Falcon support from https://github.com/SAP/cf-python-logging-support/tree/master/sap/cf_logging/django_logging and https://github.com/SAP/cf-python-logging-support/tree/master/sap/cf_logging/falcon_logging to json_logging

**Comment by danizen** [comment]
Is it better to do this in this module, or another that uses `json_logging.register_framework_support`.


Whether in this package or another, the way to make it easy for a Django developer is to use a Django middleware and the [LOGGING_CONFIG](https://docs.djangoproject.com/en/2.2/ref/settings/#logging-config) setting which can pre-process the dictionary usually passed to `logging.dictConfig`.

This would look like this in a Django settings file:

```python
MIDDLEWARE = [
    'logging_config.framework.django.RequestResponseMiddleware',
    # ... rest of middleware
]

LOGGING_CONFIG='json_logging.framework.django.config_logging'
```

And then the `LOGGING_CONFIG `callable would do the required wiring:

```python
def configure_logging(logging_dict):
    logging.dictConfig(logging_dict)
    json_logging.ENABLE_JSON_LOGGING = True
    json_logging.init_django()
```

**Comment by thangbn** [comment]
@danizen yeah, use can leverage the **register_framework_support** to add django support. PR is welcomed!

**Comment by danizen** [comment]
@thangbn, I have a broken wrist right now.  I code *slow* until healed.  But wait a month or so.  

I've also been wondering about the correlation id, and auto headers added in the cloud.  If you are running behind an AWS ALB, it will send [X-Amzn-Trace-Id](https://docs.aws.amazon.com/elasticloadbalancing/latest/application/load-balancer-request-tracing.html).  But this has an internal structure.  I'm not sure whether this module makes a requirement on the structure of the Correlation ID header (e.g. UUID or something).

**Comment by thangbn** [comment]
@danizen sure that is something i can look into, i also plan to support custom list of headers to be logged together with correlation-id (e.g trace id,  span id used by opentracing)

**Comment by kshpytsya** [comment]
@thangbn , @danizen are there any plans on completing this any time soon, or should I rather try tackling this myself from scratch?

**Comment by thangbn** [comment]
@kshpytsya i really dont have time to to this, you are very welcome to create a PR, it should relatively straight forward to do. Thanks in advance.

**Comment by obvionaoe** [comment]
Hi, is this still being worked on? I'd love to contribute as I really need Django support but I'm kind of a python beginner, so I don't even have a clue where to start...

**Comment by bobbui** [comment]
> Hi, is this still being worked on? I'd love to contribute as I really need Django support but I'm kind of a python beginner, so I don't even have a clue where to start...

Feel free to submit it, we can all work together. no worries
